make_report.py

```diff
@@ -62,19 +83,22 @@ def generate_report_text(input_report_file: str, input_photo_dir: str, year: int
 
     report_paragraph_contents: List[str] = [p.strip() for p in report_text.split('\n') if len(p.strip()) > 0]
 
-    photo_names = [f for f in sorted(os.listdir(input_photo_dir)) if "Small" not in f]
+    photo_names = [f for f in sorted(os.listdir(input_photo_dir)) if "Small" not in f and not f.endswith(".json")]
```

Comment:
Actually shouldn't be used in `upload_to_website` since we want to include the small images, but want to exclude the `.json` file. So `upload_to_website` does need changing but in a different way!

Reply:
I could stick in the `captions.json` file in the parent directory of the images directory?

Normally I create the imgs directory of a subdirectory of the one that contains the report file, but I don't know if that's common usage?

Reply:
I also usually do that. I normally have
```
- some_directory
-- report.txt
-- images
--- X.png
--- Y.png
```

Actually I'm wrong. `process_photos` outputs the resized photos to a temporary directory, which is what gets passed to `upload_to_website` so actually it's that temporary directory which contains all the stuff to upload, so `captions.json` won't pollute it. I think the `"Small" not in f` is actually redundant and was probably left over from before I started writing to a temporary directory. So I think what you've done does work, and we can leave it where it is.